flexmeasures/api/v3_0/sensors.py

```diff
@@ -1575,13 +1622,170 @@ def trigger_forecast(self, id: int, sensor: Sensor, **params):
+          description: |
+            Fetch the results of a previously triggered forecasting job.
+
+            While the forecasting job is still running, this endpoint returns its
+            current status. Once the job has completed successfully, the endpoint
+            returns the generated forecast.
+
+            The returned forecast represents the most recent belief per event and
+            covers the period for which forecasts were generated.
+          security:
+            - ApiKeyAuth: []
+          parameters:
+            - in: path
+              name: id
+              required: true
+              description: ID of the sensor for which the forecast was triggered.
+              example: 5
+              schema:
+                type: integer
+            - in: path
+              name: uuid
+              required: true
+              description: UUID of the forecasting job returned by the trigger endpoint.
+              example: b3d26a8a-7a43-4a9f-93e1-fc2a869ea97b
+              schema:
+                type: string
+          responses:
+            200:
+              description: Forecast job status or results
+              content:
+                application/json:
+                  schema:
+                    type: object
+                    properties:
+                      status:
+                        type: string
+                        enum: ["PENDING", "RUNNING", "FAILED", "FINISHED"]
+                      start:
+                        type: string
+                        format: date-time
+                        description: Start time of the forecast values (ISO 8601).
+                      end:
+                        type: string
+                        format: date-time
+                        description: End time of the forecast values (ISO 8601).
+                      resolution:
+                        type: string
+                        description: Resolution of the forecast values as an ISO-8601 duration.
+                      unit:
+                        type: string
+                        description: Unit of the forecast values.
+                      values:
+                        type: array
+                        items:
+                          type: number
+                        description: Forecast values.
+                  examples:
+                    finished:
+                      summary: Finished forecasting job
+                      value:
+                        start: "2025-10-15T00:00:00+01:00"
+                        end: "2025-10-15T04:00:00+01:00"
+                        resolution: "PT1H"
+                        unit: "kW"
+                        values: [1.2, 1.5, 1.4, 0.8]
+            401:
+              description: UNAUTHORIZED
+            403:
+              description: INVALID_SENDER
+            422:
+              description: UNPROCESSABLE_ENTITY
+          tags:
+            - Sensors
+        """
+
+        try:
```

Comment:
This try/except block is casting a very wide net. in `get_schedule`, we are using try/excepts in a more focused way, which I think is preferable.